src/utils/CiceroMarkToOOXML/index.js

```diff
@@ -29,12 +29,14 @@ const insertSoftBreak = async context => {
   await context.sync();
 };
 
-const insertText = async (context, value) => {
-  context.document.body.insertText(value, Word.InsertLocation.end).font.set({
+const insertText = async (context, value, emphasize=false) => {
```

Comment:
Do you think there will need to be other options passed in similar to this one with emphasize?

Reply:
@irmerk yeah, I think. Can you share the latest `ciceroMark` specs so that I can refer to it and let you know more precisely?

Reply:
I think this would be the best reference, correct @jeromesimeon?

https://docs.accordproject.org/docs/markup-commonmark.html

Reply:
Note that this has not been updated since the new parser yet.